### PR TITLE
Add Spanish translation changes by @GamerZPlay268

### PR DESCRIPTION
**Pull request by rugk**
Fixes https://github.com/rugk/awesome-emoji-picker/issues/102

Thanks @GamerZPlay268

### CODE REVIEW

**Comment by rugk** [comment]
Hi @GamerZPlay268,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Please subscribe to #27  to get notified for new updates on translations etc. :smiley:

File: src/_locales/es/messages.json [hunk 1]
```diff
@@ -2,463 +2,601 @@
...
+    "hash": "31a86798742bc224e67e9ebaebdd648f"
   },
   "emojiMartCategoryCustom": {
     "message": "Personalizado",
-    "description": "Localisation of an emoji category option of emoji-mart, the emoji-picker. See: https://github.com/missive/emoji-mart#i18n"
+    "description": "Localisation of an emoji category option of emoji-mart, the emoji-picker. See: https://github.com/missive/emoji-mart#i18n",
+    "hash": "1685452bc68395b620b0ce1fb9b76da5"
   },
   "emojiMartCategoriesLabel": {
     "message": "Categorías de emojis",
```
Comment (elamperti) [inline src/_locales/es/messages.json:534]: This pluralization should be fixed as well (my bad!)

File: src/_locales/es/messages.json [hunk 2]
```diff
@@ -2,463 +2,601 @@
...
   "translatorCredit": {
     "message": "Esta extensión ha sido traducida al español por $TRANSLATORS$.",
     "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
     "placeholders": {
-      "translators": {
-        "content": "$1",
-        "example": "<a href=\"https://github.com/rugk/\">@rugk</a>"
-      }
-    }
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
```
[the remark this answers is not in the document]
Comment (rugk) [reply src/_locales/es/messages.json:392]: We have a guideline for that: https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on

I'm now changing it to that one. I also have no idea why my name is there. :sweat_smile:

[the remark this answers is not in the document]
Comment (rugk) [reply src/_locales/es/messages.json:392]: And BTW you are already linked, the line you quoted is just an example.

Actually ou both seem to be named in `translatorUsername` properly.
I just fixed the link now.

**Comment by rugk** [comment]
So as everything has been done here and there was no reaction, I've applied as best as I could without speaking that language and so the changes are now merged.

If you want to improve them, feel free to have a look at the translation tool again and help to e.g. fix missing translations or updated ones, etc.

**Pull request merged**